packages/celotool/src/cmds/account/verify.ts

```diff
@@ -64,36 +44,35 @@ export const handler = async (argv: VerifyArgv) => {
 }
 
 async function verifyCmd(argv: VerifyArgv) {
-  const web3 = new Web3('http://localhost:8545')
+  const kit = newKit('http://localhost:8545')
+  const account = (await kit.web3.eth.getAccounts())[0]
+  kit.defaultAccount = account
```

Comment:
I think we should consider having a `newKit(url, { defaultAccount})` constructor